test_runner/regress/test_explain_with_lfc_stats.py

```diff
@@ -0,0 +1,84 @@
+import os
+
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import NeonEnv
+
+
+def test_explain_with_lfc_stats(neon_simple_env: NeonEnv):
+    env = neon_simple_env
+
+    cache_dir = os.path.join(env.repo_dir, "file_cache")
+    os.mkdir(cache_dir)
```

Comment:
Let's use `Path` here (it requires to add `from pathlib import Path`):

```suggestion
    cache_dir = env.repo_dir / "file_cache"
    cache_dir.mkdir()
```

Reply:
ok - actually I had copied that line from another test ;-)